src/girouette/tw/border.cljc

```diff
@@ -2,7 +2,17 @@
   (:require [girouette.tw.common :refer [value-unit->css]]))
 
 (def components
-  [{:id     :border
+  [{:id     :border-radius
+    :rules  "
+    border-radius = <'rounded'>  (<'-'> direction (side)?)? (<'-'> size)?
+    side = 'l' | 'r'
+    size = none | full | media-query-min-width
```

Comment:
`side` and `size` may conflict with other rules in the Instaparse, now or later.
I suggest to prefix names to reduce such collisions, like `border-radius-side` and `border-radius-size`.